dashboard/menu/dashboard/pagecache/snapshotsettings.md

```diff
@@ -0,0 +1,19 @@
+# Settings of individual Snapshots
+
+These options can be set for each Snapshot individually by clicking on their respective rows in the table and choosing the appropriate tab of the side menu that appears.
+
+## Settings
+
+Here, you can change the name of the chosen Snapshot. You can also restrict the scope of the Snapshot using the Content Matching text boxes. These accept wildcard-compatible path specifications that set whether or not the snapshot can be used on the page(s) in question. The checkboxes allow you to set what part(s) of the site to serve from the snapshot (e.g. you can set up to cache HTML and JS/CSS but not the Images if your client wants to avoid bleedthrough on normal text but doesn’t mind it in case of images such as logos). You can also see when the Snapshot was last updated (or built) using a Scan.
```

Comment:
These are not _parts_, more like components of a site.

Reply:
Changed parts to components in the latest commit.